lightning-background-processor/src/lib.rs

```diff
@@ -150,6 +150,11 @@ impl BackgroundProcessor {
 	/// uploading to one or more backup services. See [`ChannelManager::write`] for writing out a
 	/// [`ChannelManager`]. See [`FilesystemPersister::persist_manager`] for Rust-Lightning's
 	/// provided implementation.
+	/// 
+	/// `persist_graph` is responsible for writing out the [`NetworkGraph`] to disk, and/or
+	/// uploading to one or more backup services. See [`ChannelManager::write`] for writing out a
```

Comment:
Don't think we want to suggest people back up the network graph, its all public data.